modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/podcast/EpisodeManagerImpl.kt

```diff
@@ -89,31 +89,31 @@ class EpisodeManagerImpl @Inject constructor(
     private val userEpisodeDao = appDatabase.userEpisodeDao()
 
     override suspend fun findEpisodeByUuid(uuid: String): BaseEpisode? {
-        val episode = findByUuidSuspend(uuid)
+        val episode = findByUuid(uuid)
         if (episode != null) {
             return episode
         }
 
         return userEpisodeManager.findEpisodeByUuid(uuid)
     }
 
-    override fun findByUuid(uuid: String): PodcastEpisode? {
-        return episodeDao.findByUuid(uuid)
-    }
+    override suspend fun findByUuid(uuid: String): PodcastEpisode? =
+        episodeDao.findByUuid(uuid)
```

Comment:
I changed it so that `findByUuid` is now the suspended function because I felt like it makes sense to make the preferred, default function the one with the shortest name. wdyt?

Reply:
I like this change too.